donate.dd

```diff
@@ -0,0 +1,91 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(H4 The $(LINK2 $(ROOT_DIR)foundation.html, D Language Foundation) is an organization devoted to advancing
+     open source technology related to the D programming language.
+)
+
+The D Language foundation:
+
+$(COMMENT keep this list in sync with foundation.dd)
+$(UL
+    $(LI holds and defends the copyright and other intellectual property rights for the D programming language.)
+    $(LI provides resources for furthering the development and adoption of the D programming language.)
+    $(LI awards $(LINK2 http://dlang.org/blog/2016/12/05/the-d-language-foundations-scholarship-program, scholarships)
+        to highly skilled students.)
+    $(LI organizes the yearly $(HTTP dconf.org, D conference).)
+    $(LI pays for hardware costs
+        (e.g. this website, $(LINK2 http://dlang.org/download.html, dmd downloads),
+        the $(LINK2 https://tour.dlang.org, D tour) and more))
+    $(LI is an accepted
+        $(HTTPS summerofcode.withgoogle.com/organizations/5078256051027968/,
+        Google Summer of Code organization) and hosted four projects during
+        the summer of 2016.
+    )
+)
+
+$(H3 How can I donate?)
+
+We welcome contributions from both organizations and individuals. Your help is highly appreciated.
+
+$(DONATE_ITEM Donate through PayPal, paypal,
+
+    Click the PayPal button to make a donation to the D Language Foundation via PayPal:
+    $(BR) $(BR)
+
+    <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
+    <input type="hidden" name="cmd" value="_s-xclick" />
+    <input type="hidden" name="hosted_button_id" value="6DV3SYW3A5ZPG" />
```

Comment:
OK. There are a few more pages:

* Return URL (after the donation is made
* Canceled checkout URL
* Return to Merchant Link
* URL for IPN messaging - no idea what that is

For now I didn't customize any of these. I got this code:

```html
<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
<input type="hidden" name="cmd" value="_s-xclick" />
<input type="hidden" name="hosted_button_id" value="K8PLNUP7276BQ" />
<input type="image" src="https://www.paypal.com/en_US/i/btn/btn_donateCC_LG.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate" />
<img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1" />
</form>
```

and the following url: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=K8PLNUP7276BQ